web/src/components/Incidents/AlertsChart/AlertsChart.jsx

```diff
@@ -74,9 +81,9 @@ const AlertsChart = ({ alertsData = [], chartDays }) => {
               }
               domainPadding={{ x: [30, 25] }}
               legendData={[
-                { name: 'Critical', symbol: { fill: global_danger_color_100.var } },
-                { name: 'Info', symbol: { fill: global_info_color_100.var } },
-                { name: 'Warning', symbol: { fill: global_warning_color_100.var } },
+                { name: 'Critical', symbol: { fill: '#c9190b' } },
```

Comment:
Are these colors compatible with dark mode?

Reply:
I think colors will change anyway with https://issues.redhat.com/browse/OU-634 (see comments)

Reply:
Then at least make sense to define this as constants and share them to avoid mistyping or using a different color in different places.

Reply:
@Fewwy ok, please fix the issue above. Maybe in the future we can try to include changes for unrelated tasks in a specific PR so they don't block bigger changes.